.travis.yml

```diff
@@ -28,9 +28,8 @@ matrix:
         - RUST_BACKTRACE=1 LLVM_CONFIG_PATH=llvm-3.8-config
       before_script:
         - rustup target add x86_64-fortanix-unknown-sgx x86_64-unknown-linux-musl
-        - cd intel-sgx
       script:
-        - cargo test --verbose --locked --all --exclude sgxs-loaders && [ "$(echo $(nm -D ../target/debug/sgx-detect|grep __vdso_sgx_enter_enclave))" = "w __vdso_sgx_enter_enclave" ]
```

Comment:
I don't understand why this change is only needed now and not in a previous PR

Reply:
It's not strictly needed now. Before the scripts, the working directory is changed to `intel-sgx`. That works but would be a bit weird as we'd have to call `../doc/generate-api-docs.sh`